src/core/electrostatics_magnetostatics/common.hpp

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
i guess we want an header guard [LGTM](https://lgtm.com/projects/g/espressomd/espresso/rev/pr-22060e5ec6266b3319a9a9ed0f7c3443972b76ef)

Reply:
indeed, I'll add one